build/make/makebin.linux

```diff
@@ -7,7 +7,7 @@
 # Linux (GCC)
 #
 
-CFLAGS+=-fPIC -Wall -O3 -ansi -D_GNU_SOURCE -D_LINUX
+CFLAGS+=-fPIC -Wall -O3 -g -ansi -D_GNU_SOURCE -D_LINUX
```

Comment:
Is the debug flag necessary to propagate into the develop branch and therefore an eventual release?  That seems like something that should remain local for testing purposes.

Reply:
I agree, this should probably be removed.